org-cyf-itp/content/data-flows/success/index.md

```diff
@@ -1,15 +1,37 @@
 +++
 title = "Success"
-description = "How do we know if we have completed this module?"
+description = "How do we know if we have completed Data Flows?"
 layout = "success"
 emoji= "✅"
 menu_level = [ "module" ]
 weight = 11
 [[objectives]]
-1="Describe how data is mapped to a UI"
-2="Break down a UI into components"
-3="Reason about code and time"
-4="Define, use, update, and derive state"
-5="Solve at least 6 5kyu problems in Codewars"
-6="Solve up to level 30 in the Bandit repeatedly"
+1="Link to your PR for [Array Destructuring](https://github.com/CodeYourFuture/Module-Data-Flows/issues/24), which you have had reviewed and responded to feedback."
+2="Link to your PR for your [Book Library Project](https://github.com/CodeYourFuture/Module-Data-Flows/issues/31)  which you have had reviewed and responded to feedback."
+3="Worked out your LinkedIn [Social Selling Index](https://github.com/CodeYourFuture/Module-Data-Flows/issues/12), and posted the actions you will take to improve this"
+4="Link to your PR for [Javascript challenges](https://github.com/CodeYourFuture/Module-Data-Flows/issues/21), which you have had reviewed and responded to feedback."
 +++
+
+
+
+### 🎯 You've achieved your learning objectives if you can:
+
+```objectives
```

Comment:
feels like these are more limited in scope than the original ones. In particular 4="Define, use, update, and derive state" seems important to try to keep. What do you think?